### PR TITLE
[docs] new blogpost about community

### PR DESCRIPTION
**Pull request by bjornalm**
## What changes were proposed in this pull request?

- new post

## How was this patch tested?

- (Please explain how this patch was tested. Ex: unit tests, manual tests)
- (If this patch involves UI changes, please attach a screen-shot; otherwise, remove this)

Please review [Hue Contributing Guide](https://github.com/cloudera/hue/blob/master/CONTRIBUTING.md) before opening a pull request.

### CODE REVIEW

File: docs/gethue/content/en/posts/2023-03-01-hue-community-2023.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  - default
+categories:
+  - community
+  - development
+
+---
+
+Hello everyone!
+
+Last year was a bit slow for the open source community around Hue but we are planning to change that going forward in 2023. There will be more frequent releases, improved issue handling process and a wider initiative to engage more people outside of Cloudera. We want the Hue community to be vibrant, useful and inclusive and we will try to lower the threshold for those who want to engage by simplifying, encouraging and rewarding contributions in all forms.
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2023-03-01-hue-community-2023.md:49]: Last year was a bit slow for the open-source community around Hue, but we plan to change that in 2023. There will be more frequent releases, an improved issue-handling process, and a larger initiative to engage more people outside Cloudera. We want the Hue community to be vibrant, active, and inclusive. We will try to lower the threshold for those who wish to engage by simplifying, encouraging, and rewarding contributions in all forms.

File: docs/gethue/content/en/posts/2023-03-01-hue-community-2023.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+We start off these improvements by establishing a process for how to handle incoming issues for our [Github repository](https://github.com/cloudera/hue/issues). Although Hue is open source and used in the Hadoop services of all the major cloud providers, it is still mainly developed and maintained by the Hue core team at Cloudera. Upstream code changes and releases for the open source community coexist with more frequent downstream releases like [Cloudera Data Platform (CDP)](https://www.cloudera.com/products/cloudera-data-platform.html) and we always try to balance these two sides of Hue. When it comes to handling incoming issues like bug reports, requests for enhancements or questions, our goal is to increase the amount that gets resolved by the help of external committers and experts, but we also don’t want issues to go stale in lack of community engagement. Therefore we have put in place a process to make sure that:
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2023-03-01-hue-community-2023.md:51]: We start these improvements by establishing a process for handling incoming issues for our [Github repository](https://github.com/cloudera/hue/issues). Although Hue is open source and used in the Hadoop services of all the major cloud providers, it is still mainly developed and maintained by the Hue core team at Cloudera. Upstream code changes and releases for the open source community coexist with more frequent downstream releases like [Cloudera Data Platform (CDP)](https://www.cloudera.com/products/cloudera-data-platform.html), and we always try to balance these two sides of Hue. For handling incoming issues like bug reports, requests for enhancements, or questions, our goal is to increase the amount that gets resolved with the help of external committers and experts. But we also don't want issues to go stale in a lack of community engagement. Therefore, we have put in place a process to make sure that:

File: docs/gethue/content/en/posts/2023-03-01-hue-community-2023.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+  * Incoming issues are filed using improved templates to speed up resolution times.
+  * The core team looks at all incoming issues and labels them as either “bug” or “configuration”. 
+    Incoming issues that are in fact questions get transformed to Github discussions.
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2023-03-01-hue-community-2023.md:55]: * Incoming issues of the type "questions" get transformed into GitHub discussions.

File: docs/gethue/content/en/posts/2023-03-01-hue-community-2023.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  * For configuration issues that can be resolved through the issue comments we hope for contribution by community members, but there will always be a core team developer responsible for trying to find a solution.
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2023-03-01-hue-community-2023.md:56]: * Configuration issues are resolved through the issue comments with contributions by the community members as much as possible. However, a core team developer will be responsible for finding a solution.

File: docs/gethue/content/en/posts/2023-03-01-hue-community-2023.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+  * For bugs that require code changes a member of the core team will own the issue and track it internally so that it isn’t forgotten. We keep the open source Github issue unassigned until work starts in case someone from the community would like to create a pull request for it.
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2023-03-01-hue-community-2023.md:57]: * A core team member owns the issue and tracks it internally for bugs that require code changes. We will keep the open-source GitHub issue unassigned until work starts or until someone from the community creates a pull request for it.

File: docs/gethue/content/en/posts/2023-03-01-hue-community-2023.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+  * Enhancement and feature requests will be brought to the attention of the core team and product owners at Cloudera, and might end up on the internal release planning in which case we will let the reporter know. But in general we will encourage that our open source community contributes with the implementation of new features.
```
Comment (pychheda) [inline docs/gethue/content/en/posts/2023-03-01-hue-community-2023.md:58]: * Enhancement and feature requests are brought to the attention of the core team and product owners at Cloudera. We will inform the reporter if the enhancement or feature request has been included in the internal release planning. However, we encourage our open-source community to contribute to developing new features.

**Review by pychheda — COMMENTED** [review]
Nice post! All my suggestions are to simplify reading.
Thanks for running this by me.

**Comment by bjornalm** [comment]
> Nice post! All my suggestions are to simplify reading. Thanks for running this by me.

Great, thanks for the feedback. I updated the PR.

**Review by pychheda — APPROVED** [review]
LGTM.

**Review by JohanAhlen — APPROVED** [review]
Super nice!

**Review by nidhibhatg — COMMENTED** [review]
LGTM

**Pull request merged**